Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax = docker/dockerfile:experimental
 
 ARG KUBECTL_VERSION=1.28.4
-ARG HELM_VERSION=3.13.3
+ARG HELM_VERSION=3.14.0
```

Comment:
This helm version leaves out of support Kubernetes `1.25` which is actively supported by Okteto: https://helm.sh/docs/topics/version_skew/#supported-version-skew

Isn't there available a patch for `3.13.X`?

Reply:
No, 3.13.3 is the latest 3.13 release. Also I've been checking and they don't continue patching previous helm versions so I've decided to revert the change until we don't support k8s 1.25